modules/controller/command_parser.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Dict, Tuple
+
+from modules.controller.commands.classify_command import ClassifyCommand
+from modules.controller.commands.command import Command
+
+
+##  Class for parsing strings to a command
+from modules.controller.commands.collector_command import CollectorCommand
+from modules.controller.commands.label_command import LabelCommand
+from modules.controller.commands.quit_command import QuitCommand
+from modules.controller.commands.train_command import TrainCommand
+from modules.exception.excpetions import IllegalArgumentException
+
+
+class CommandParser:
+
+    _valid_commands: Dict[str, Command.__class__] = {
+        "collect": CollectorCommand,
+        "labeler": LabelCommand,
+        "train": TrainCommand,
+        "classify": ClassifyCommand,
+        "quit": QuitCommand,
+    }
+
+    ##  Parses a string to a command
+    #
+    #   @param input_string the string that should be parsed to a command
+    @staticmethod
+    def parse_input(input_string: str) -> Command:
+        arg_list = input_string.split(" ")
```

Comment:
How is the input of the variable? Aren't there commas in between the different input vairables? Or is it always just a blank?